discovery-infra/assisted_service_api.py

```diff
@@ -129,6 +137,14 @@ def set_hosts_roles(self, cluster_id, hosts_with_roles):
             cluster_id=cluster_id, cluster_update_params=hosts
         )
 
+    def set_pull_secret(self, cluster_id, pull_secret):
+        log.info(
+            "Setting pull secret for cluster %s", cluster_id
+        )
+        update_params = models.ClusterUpdateParams(pull_secret=pull_secret)
+        return self.client.update_cluster(
```

Comment:
where we are updating cluster? in assisted-service? so it is the same cluster we can take all data from it?

Reply:
this is the day2 cluster, not the installed cluster